seed/challenges/00-getting-started/getting-started.json

```diff
@@ -128,13 +128,13 @@
             [
               "//i.imgur.com/UVB9hxp.jpg",
               "Ein Foto von einem Camper in einem Cafe der an freeCodeCamp Projekten arbeitet.",
-              "Wir glauben dass Menschen am Besten beim Tun lernen. Deswegen wirst du die meiste Zeit damit beschäftigt sein an tatsächlichen Projekten zu arbeiten. Wir werden dir eine Liste an Anforderungen (agile User Stories) geben und du wirst dich damit auseinander setzen diese zu erfüllen.",
+              "Wir glauben, dass Menschen am Besten durch aktives Anwenden lernen. Deswegen wirst du die meiste Zeit damit beschäftigt sein an tatsächlichen Projekten zu arbeiten. Wir werden dir eine Liste an Anforderungen (agile User Stories) geben. Deine Aufgabe ist es, diese zu erfüllen.",
```

Comment:
How about this?
` Dazu geben wir dir eine Liste an Anforderungen (agile User Stories) und deine Aufgabe ist es, diese umzusetzen.",`